balancer/pickfirst/pickfirst_ext_test.go

```diff
@@ -376,12 +376,12 @@ func (s) TestPickFirst_StickyTransientFailure(t *testing.T) {
 			},
 		}),
 	}
-	cc, err := grpc.Dial(lis.Addr().String(), dopts...)
+	cc, err := grpc.NewClient(lis.Addr().String(), dopts...)
 	if err != nil {
-		t.Fatalf("Failed to dial server at %q: %v", lis.Addr(), err)
+		t.Fatalf("Failed to create new client server at %q: %v", lis.Addr(), err)
```

Comment:
Please fix the grammar. We can also remove the server address because NewClient doesn't create a transport immediately, so it unlikely that the server had anything to do with the failure.
```
t.Fatalf("Failed to create new client: %v", err)
```

Reply:
done please check